src/registered.js

```diff
@@ -59,5 +62,23 @@ module.exports = function (request, h) {
     })
     .write();
 
-  return h.view('confirmed', null, { layout: 'index' });
+
+  const apiPasetoKey = await getPasetoKey();
+  const apiPayload = {
+    email: pec,
+    codeHosting: [{
+      url: url
+    }],
+  };
+
+  fetch(config.apiURL + '/publishers', {
+    method: 'POST',
+    body: JSON.stringify(apiPayload),
+    headers: {'Content-Type': 'application/json', 'Authorization': 'Bearer ' + apiPasetoKey}
+  }).then(res => {
+    console.log(res.json());
```

Comment:
we should be able to manage possible errors and notify them to the user

Reply:
Yes, of course.
Actually I'm in vacation, could @bfabio add the small catch part to that, so we can in the meantime deploy and try that?
This could lead to a better timings for the upcoming demo.
Thanks!